src/documentation/markdown.rs

```diff
@@ -382,4 +418,107 @@ And a line, or a rule."##;
             });
         assert_eq!(v.first(), Some(&(12..14)));
     }
+
+    #[test]
+    fn markdown_reduction_mapping_inline_link() {
+        const MARKDOWN: &str = r#" dyrck [I'm an inline-style link](https://www.google.com) artic"#;
+        const PLAIN: &str = r#"dyrck I'm an inline-style link artic"#;
```

Comment:
```suggestion
        const PLAIN: &str = r#"prefix I am feeling ducky postfix"#;
```

Reply:
`prefix` is not a misspelled word here. My idea was to have two misspelled words one before the [inline text] and after. Do you have any preferential sets of words for examples?

Reply:
I prefer wording around privacy aware companies rather than known big $ corp. I also prefer shorter tests where possible, showing precisly one variant, except for end 2 end tests.

It becomes less cluttered once you have a macro for it :) and adding more test cases becomes simpler

Reply:
do you have something similar already in the cargo-spellcheck code? maybe it will help me to check it out :)

Reply:
I will keep that on mind next time ;)